test/test_ao.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+from torch.testing._internal.common_utils import run_tests
```

Comment:
Should we call this file test_ao_sparsity.py so that we eventually maintain separate top level files for quantization and sparsity?

Reply:
Sure, that's not a bad plan